build.rs

```diff
@@ -1 +1,31 @@
-fn main() {}
+#[allow(unused_imports)]
+use std::{path::PathBuf, process::Command};
+
+#[allow(dead_code)]
+fn main() {
```

Comment:
This logic should be a part of frontend `build.rs`. Frontent artifacts should be copied to `target` dir by its own build script, not the top level one. Actually, we don't need a top level script at all